src/compile/data/transforms.ts

```diff
@@ -48,6 +50,48 @@ export class FilterNode extends DataFlowNode {
   }
 }
 
+export class GeoJSONNode extends DataFlowNode {
+  public clone() {
+    return new GeoJSONNode(duplicate(this.fields), this.geojson, this.signal);
+  }
+
+  public static make(model: ModelWithField): GeoJSONNode {
+    const geo = model.reduceFieldDef((rel, def, channel) => {
+      if ((contains([X, Y], channel) && contains([LATITUDE, LONGITUDE], def.type))
```

Comment:
Yes. that doesn't _really_ make any sense in Vega-Lite, but the GeoJSON transform does support that, so we might as well compile it correctly.